hw/ip/aes/pre_syn/tcl/yosys_run_synth.tcl

```diff
@@ -33,7 +33,7 @@ yosys "attrmap -tocase keep -imap dont_touch=\"yes\" keep=1 -imap dont_touch=\"n
 # Place keep_hierarchy contraints on relevant modules to prevent aggressive synthesis optimzations
 # across the boundaries of these modules.
 yosys "hierarchy -check -top $lr_synth_top_module"
-yosys "setattr -mod -set keep_hierarchy 1 *prim_xilinx_buf*"
+yosys "setattr -mod -set keep_hierarchy 1 *prim_xilinx*"
```

Comment:
if you want to more accurately model this, you could place this constraint only on the prims that we actually constrain:
https://github.com/lowRISC/opentitan/blob/72643363e818f11a292d87913b7b3acc0e002714/hw/syn/tools/dc/gtech-constraints.sdc#L12-L18

otherwise it will affect other prims as well (granted, it does not apply to all prims since it is technology specific, but still).

Reply:
Oh, that's a good point. Right now, and for AES it's okay because in AES we just use `xor`, `buf`, `flop`, and `flop_en` but no other Xilinx/generic prims like memories etc. But I should fix this at some point.